.npmignore

```diff
@@ -23,3 +23,10 @@ config
 .travis.yml
 CHANGELOG.md
 README.md
+
+*venv
+*__pycache__
+.circleci
+.vscode
+.github
+dist
```

Comment:
dist should be part of the npm package

Reply:
Does `dist/` contain any `.js` file? I was under the impression only the generated wheels goes there

Reply:
does `dist` contain any JS file? I thought only the wheels go there (might be wrong)

Reply:
Actually, might be a better idea to use the `files` field: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files

Reply:
You don't want to ignore  the js files that should be part of the npm package. no need to add then in package.json in files.